app/assets/javascripts/govuk_publishing_components/lib/auto-track-event.js

```diff
@@ -0,0 +1,31 @@
+// migrated from govuk_frontend_toolkit
```

Comment:
This is only used in old publishing tools via [govuk_admin_template which has its own copy of this code](https://github.com/alphagov/govuk_admin_template/blob/0dfd5cffe8ec6ffa2d29767531f1e64826773b7e/app/assets/javascripts/govuk-admin-template/modules/auto_track_event.js).

Reply:
I was wrong with this, [`wrapper layout in static` still uses it as a tertiary dependency](https://github.com/alphagov/static/blob/976e5659c8c199ce9a7a79425eae0474dc880a5c/app/assets/javascripts/application.js#L4) that does automagic tracking in `frontend` 🤦